roles/install_klipper/tasks/recore.yml

```diff
@@ -218,18 +208,3 @@
     mode: '0755'
     owner: debian
     group: debian
-
-- name: Recore - make sure the sudoers.d/debian file exists
-  file:
-    path: /etc/sudoers.d/debian
-    owner: root
-    group: root
-    state: touch
-
-- name: Recore - create folder for klipper logs in /var/log
-  file:
-    path: /var/log/klipper_logs
```

Comment:
This looks like a problem on OctoPrint. Klipper will not start: 
FileNotFoundError: [Errno 2] No such file or directory: '/var/log/klipper_logs/klippy.log'

Reply:
Will revert this. I hadn't thought of the need to store the log in zram. I'll also add symlinks in the linux_preparation stage for the entire `/home/debian/printer_data/logs/` folder to be in `/var/log/` as well.